noxfiles/run_infrastructure.py

```diff
@@ -98,29 +99,31 @@ def run_infrastructure(
         )
 
     if run_create_superuser:
-        return _run_create_superuser(path, IMAGE_NAME)
+        return _run_create_superuser(path, COMPOSE_SERVICE_NAME)
 
     if run_create_test_data:
-        return _run_create_test_data(path, IMAGE_NAME)
+        return _run_create_test_data(path, COMPOSE_SERVICE_NAME)
 
 
 def seed_initial_data(
     datastores: List[str],
     path: str,
-    base_image: str,
+    service_name: str,
 ) -> None:
     """
     Seed the datastores with initial data as defined in the file at `setup_path`
     """
     _run_cmd_or_err('echo "Seeding initial data for all datastores..."')
     for datastore in datastores:
         if datastore in DOCKERFILE_DATASTORES:
-            setup_path = f"tests/integration_tests/setup_scripts/{datastore}_setup.py"
+            setup_path = (
+                f"tests/ops/integration_tests/setup_scripts/{datastore}_setup.py"
```

Comment:
can we store `tests/ops` as a global const `ops_test_dir` that can be shared here and in  our `ci_nox.py` too?